cypress/integration/transfer/reorder-with-buttons.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Reorder items in the selected list
+
+  Background:
+    Given reordering of items is enabled
+
+# I created two scenarios for up and down reordering because I wanted to incorporate a check for 'is first' or 'is last' and it seemed overly ambiguous to write this in a single scenario. Or, is it enough that I have included the 'Disable move buttons' scenarios at the bottom of this file, so that doesn't need to be handled in the basic scenarios at the top?
+
+  Scenario: Reorder items upwards using buttons
+    Given a selected list with one or more items
+    And a highlighted item in the selected list
+    And the highlighted item is not the first item in the list
+    When the 'move up' button is clicked
+    Then the highlighted item moves up one place in the selected list
```

Comment:
I think we can modify this one and and include what happens when the `move up` is pressed when the first item is highlighted.

```feature
Scenario Outline: Reorder items upwards using buttons
    Given a selected list with one or more items
    And the <previous> item is highlighted
    When the 'move up' button is clicked
    Then the highlighted item is moved to the <next> place

    Examples:
        | previous | next |
        | 1        | 1    |
        | 2        | 1    |
        | 3        | 2    |
```

Reply:
Really cool use of a Scenario Outline!